source/jormungandr/tests/places_tests.py

```diff
@@ -363,6 +369,23 @@ def test_places_has_no_distance(self):
         assert len(places) == 1
         assert 'distance' not in places[0]
 
+    def test_places_to_valide_any_beta(self):
```

Comment:
Not sure between
```suggestion
    def test_places_to_valid_any_beta(self):
```
or
```suggestion
    def test_places_to_validate_any_beta(self):
```

Reply:
:+1: